test/sqllogictest/materialized_views.slt

```diff
@@ -0,0 +1,375 @@
+# Copyright Materialize, Inc. and contributors. All rights resemved.
```

Comment:
This is just a renamed `test/sqllogictest/recorded_views.slt` with all occurrences of "recorded view" replaced. Looks like git isn't able to pick this up.